frontend/src/lib/hooks/api/kafkaConnect.ts

```diff
@@ -92,13 +92,19 @@ export function useConnectorTasks(props: UseConnectorProps) {
     }
   );
 }
-export function useUpdateConnectorState(props: UseConnectorProps) {
+export function useUpdateConnectorState(clusterName: ClusterName) {
   const client = useQueryClient();
   return useMutation(
-    (action: ConnectorAction) => api.updateConnectorState({ ...props, action }),
+    ({
```

Comment:
can we use object syntax here , instead of the function positional stuff , cause that one is deprecated in v5

Reply:
fixed